src/ngraph/pass/manager.cpp

```diff
@@ -25,12 +25,18 @@
 #include "ngraph/op/reduce.hpp"
 #include "ngraph/pass/manager.hpp"
 #include "ngraph/pass/pass.hpp"
+#include "ngraph/pass/visualize_tree.hpp"
 
 using namespace std;
 using namespace ngraph;
 
 ngraph::pass::Manager::Manager()
 {
+    static const auto nevt = std::getenv("NGRAPH_ENABLE_VISUALIZE_TRACING");
```

Comment:
Would be nice to have a programmatic way to set this too.

Reply:
add a setter so visualization can be turned on and off